<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ورود</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../RC/resources.qrc">
    <normaloff>:/app-icon.jpg</normaloff>:/app-icon.jpg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ------------------------------------- */
#frameMain{
border: 3px solid gray;
}
/* ------------------------------------- */
QLabel {
font-weight: bold;
/*font-size: 15px;*/
}
/* ------------------------------------- */
QLineEdit, QComboBox {
padding: 3px;
border: 1px solid gray;
border-radius: 2px;
font-weight: bold;
/*font-size: 15px;*/
}
/* ------------------------------------- */
QPushButton{
color: #111;
border: 2px solid #555;
border-radius: 5px;
padding: 2px;
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
min-width: 80px;
min-height: 30px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
}

/* ------------------------------------- */

QToolButton{
color: rgb(0, 0, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameMain">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetMain">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageLogin">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblLogoUser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap">
             <pixmap resource="../RC/resources.qrc">:/user-login.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLineEdit" name="ledPassword">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>123456</string>
              </property>
              <property name="maxLength">
               <number>12</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ledUsername">
              <property name="text">
               <string>admin</string>
              </property>
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblUserName">
              <property name="maximumSize">
               <size>
                <width>131</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>نام کاربری</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblPassword">
              <property name="maximumSize">
               <size>
                <width>131</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>کلمه عبور</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="Space1" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>5</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnPasswordHint">
              <property name="toolTip">
               <string>یادآوری کلمه عبور</string>
              </property>
              <property name="icon">
               <iconset resource="../RC/resources.qrc">
                <normaloff>:/img/reset-password.png</normaloff>:/img/reset-password.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnLogIn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>88</width>
                <height>38</height>
               </size>
              </property>
              <property name="text">
               <string>ورود</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClose">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>88</width>
                <height>38</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>بستن</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageRegistration">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblLogoUser_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap">
             <pixmap resource="../RC/resources.qrc">:/user-login.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="btnGoToLoginPage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>رفتن به صفحه ورود</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../RC/resources.qrc">
                <normaloff>:/user-login.png</normaloff>:/user-login.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnRegister">
              <property name="text">
               <string>ثبت</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelRegistrationPage">
              <property name="text">
               <string>لغو</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QLineEdit" name="ledUsernameRegistrationPage">
              <property name="maxLength">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>نام کاربری</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>یادآوری رمز</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="ledPasswordAgainRegistrationPage">
              <property name="maxLength">
               <number>18</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="ledPasswordHint">
              <property name="maxLength">
               <number>50</number>
              </property>
              <property name="placeholderText">
               <string>یادآوری رمز عبور هنگام فراموشی</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ledPasswordRegistrationPage">
              <property name="maxLength">
               <number>18</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>رمز عبور</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>تکرار رمز</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledUsername</tabstop>
  <tabstop>ledPassword</tabstop>
  <tabstop>btnLogIn</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnPasswordHint</tabstop>
  <tabstop>ledUsernameRegistrationPage</tabstop>
  <tabstop>ledPasswordRegistrationPage</tabstop>
  <tabstop>ledPasswordAgainRegistrationPage</tabstop>
  <tabstop>ledPasswordHint</tabstop>
  <tabstop>btnRegister</tabstop>
  <tabstop>btnCancelRegistrationPage</tabstop>
 </tabstops>
 <resources>
  <include location="../RC/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ledUsernameRegistrationPage</sender>
   <signal>returnPressed()</signal>
   <receiver>ledPasswordRegistrationPage</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPasswordRegistrationPage</sender>
   <signal>returnPressed()</signal>
   <receiver>ledPasswordAgainRegistrationPage</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPasswordAgainRegistrationPage</sender>
   <signal>returnPressed()</signal>
   <receiver>ledPasswordHint</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPasswordHint</sender>
   <signal>returnPressed()</signal>
   <receiver>btnRegister</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledUsername</sender>
   <signal>returnPressed()</signal>
   <receiver>ledPassword</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogIn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
